src/commandsHandler.ts

```diff
@@ -8,9 +8,11 @@ import {
 import { configuration } from "./binary/requests/requests";
 import setState from "./binary/requests/setState";
 import { Capability, isCapabilityEnabled } from "./capabilities/capabilities";
+import handleSaveSnippet from "./saveSnippetHandler";
 
 export const CONFIG_COMMAND = "TabNine::config";
 export const STATUS_BAR_COMMAND = "TabNine.statusBar";
+export const SAVE_SNIPPET_COMMAND = "Tabnine.saveSnippet";
```

Comment:
Maybe you could align all tasks to have an identical "format"? Whichever you choose.